### ISSUE TITLE
sudo cpanm Zonemaster::LDNS Fails Debian 9

### ISSUE THREAD

**Issue by Sayuri82** [zonemaster/zonemaster-ldns#99]
Hi! I have this error when execute the command:
> cpanm Zonemaster::LDNS 

I have made the previews steps. In debian 8 i didn't have problems.
```
Unpacking Zonemaster-LDNS-2.0.1.tar.gz
Entering Zonemaster-LDNS-2.0.1
Checking configure dependencies from META.yml
Checking if you have Devel::PPPort 3.19 ... Yes (3.32)
Checking if you have Devel::CheckLib 0.4 ... Yes (1.07)
Checking if you have Module::Install 1.19 ... Yes (1.19)
Checking if you have Module::Install::XSUtil 0 ... Yes (0.45)
Checking if you have ExtUtils::ParseXS 3.18 ... Yes (3.31)
Checking if you have ExtUtils::MakeMaker 6.59 ... Yes (7.10_02)
Configuring Zonemaster-LDNS-2.0.1
Running Makefile.PL
Writing ppport.h
Feature internal ldns enabled
wrong result: 'crypto'
-> N/A
-> FAIL Configure failed for Zonemaster-LDNS-2.0.1.
```
I tried to execute it with
> --configure-args="--no-ed25519"

like you said at post #99 
but it was the same

Thanks.

**Comment by matsduf** [comment]
The support for Ed25519 will be added in the upcoming release, which you will find in the develop branch. For version 2.0.1, which is part of the Zonemaster [v2019.1.1] release, Ed25519 is not supported.

You write that you have no problem in Debian 8. In what OS do you get the issue?

[v2019.1.1]: https://github.com/zonemaster/zonemaster/releases/tag/v2019.1.1

**Comment by Sayuri82** [comment]
At Debian 9. I wrote it in the tittle, sorry

**Comment by matsduf** [comment]
> At Debian 9. I wrote it in the tittle, sorry

Thank you. I will try in Debian 9 to see what I get, and report here.

**Comment by Sayuri82** [comment]
Thank you so much

**Comment by sandoche2k** [comment]
@matsduf anyone testing on Debian 9?

**Comment by matsduf** [comment]
Not done yet. @mattias-p?

**Comment by mattias-p** [comment]
I haven't got around to it. I'm going to take it up today.

**Comment by sandoche2k** [comment]
@mattias-p any update? I don't see any issue in CentOS 7 & 8

**Comment by matsduf** [comment]
We have now released [v2.1.0] of Zonemaster-LDNS (within release [v2019.2] of Zonemaster) and we have not seen the problem reported in this issue when tested.

@Sayuri82, thank you for reporting and please create a new issue if you still see the problem.

[v2.1.0]: https://github.com/zonemaster/zonemaster-ldns/releases/tag/v2.1.0
[v2019.2]: https://github.com/zonemaster/zonemaster/releases/tag/v2019.2
